src/main/kotlin/no/nav/familie/dokument/ApiExceptionHandler.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.familie.dokument
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+
+
+@Suppress("unused")
+@ControllerAdvice
+class ApiExceptionHandler {
+
+    private val logger = LoggerFactory.getLogger(ApiExceptionHandler::class.java)
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    @ExceptionHandler(Throwable::class)
+    fun handleThrowable(throwable: Throwable): ResponseEntity<Ressurs<Nothing>> {
+        secureLogger.error("En feil har oppstått", throwable)
+        logger.error("En feil har oppstått: ${throwable.javaClass.simpleName} ")
```

Comment:
NestedExceptionUtils har en metode som heter getMostSpecificCause. Kanskje man kan bruke den i stedet for å finne feilmelding. Slik at det ikke bare står RuntimeException